#! /bin/sh /usr/share/dpatch/dpatch-run
## 002_apachectl by Adam Conrad <adconrad@0c3.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: envvars lives in /etc/apache2 in our setup.

@DPATCH@
--- apache2/support/apachectl.in	2003-05-07 13:18:19.000000000 +0100
+++ apache2/support/apachectl.in	2003-05-07 13:17:37.000000000 +0100
@@ -31,8 +31,8 @@
 HTTPD='@exp_sbindir@/@progname@'
 #
 # pick up any necessary environment variables
-if test -f @exp_sbindir@/envvars; then
-  . @exp_sbindir@/envvars
+if test -f @exp_sysconfdir@/envvars; then
+  . @exp_sysconfdir@/envvars
 fi
 #
 # a command that outputs a formatted text version of the HTML at the
